src/app/services/user.service.ts

```diff
@@ -12,25 +12,26 @@ import { UserResponse } from '../models/api-user-response';
 export class UserService {
 	private static readonly apiUrl: URL = new URL('https://api.loklak.org/api/user.json');
 	private static minified_results = true;
+	private static followers = '4';
+	private static following = '4';
```

Comment:
followers and following count has been set to 4 which reduces time in loading of sidebar user-info-box and makes space for other sections of sidebar